metadata-ingestion/src/datahub_provider/entities.py

```diff
@@ -15,6 +15,7 @@ def urn(self) -> str:
 
 @attr.s(auto_attribs=True, str=True)
 class Dataset(_Entity):
+    template_fields: ClassVar = ("platform", "name", "env",)
```

Comment:
Can you add documentation here for future reference. Something like what you have in the description.

Reply:
Added